### PR TITLE
Moved Child Footer file and added component where necessary

### PR DESCRIPTION
**Pull request by ghost**
- Renamed Footer.js to ChildFooter.js
- Moved ChildFooter.js to the common folder as it is being used across multiple pages in the child path
- Added ChildFooter component to all appropriate files so the footer would show on all child path pages
- Made some fixes to spelling errors I noticed in files while navigating through the codebase

Trello: https://trello.com/c/hPJ5tj7E

### CODE REVIEW

File: src/components/common/ParentFooter.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Link } from 'react-router-dom';
 import '../../styles/less/ParentFooter.less';
 
 export default function ParentFooter(props) {
-  // if no need to csroll go get the bottom of the page -> pass `no-scroll` by props
+  // if no need to scroll go to the bottom of the page -> pass `no-scroll` by props
```
Comment (rhiannonstanford) [inline src/components/common/ParentFooter.js:7]: Nice catch on spelling.

**Review by rhiannonstanford — APPROVED** [review]
Nice detailed footer refactor.  This will go a long ways to enhancing the organization and improving functionality of the app.  You paid careful attention to all instances and appreciate the comment clean up where necessary.  Thank you!

**Review by ethankarp — APPROVED** [review]
Impressive work Barry!

**Review by artofmayhem — APPROVED** [review]
Total rock star status on this. Nice clean up and tighten up

**Review by Yonathan-Admasu728 — APPROVED** [review]
great attention to detail in all 25 files making all the necessary changes.

**Review by milklot — APPROVED** [review]
Thanks for fixing my typos )
Good job on a other part, looks nice.

**Pull request merged**